packages/plugin-essentials/sources/commands/install.ts

```diff
@@ -27,6 +27,9 @@ export default class YarnCommand extends BaseCommand {
   @Command.String(`--cache-folder`)
   cacheFolder?: string;
 
+  @Command.Boolean(`--silent`)
+  silent?: boolean = false;
```

Comment:
Angular cli use the `--silent` option on yarn install... I added this only to bypass the problem, what should we do?